packages/core/components/Button/Button.tsx

```diff
@@ -1,9 +1,54 @@
-import Btn from '../shared-style/Btn';
+import React from 'react';
+import styled from 'styled-components';
+import { border } from './../shared-style/Border';
 
-const Button = Btn;
+export const StyledButton = styled.button<ButtonProps>`
+  background-color: ${({ theme }) => theme.colors.material};
+  color: ${({ theme }) => theme.colors.materialText};
+  padding: 6px 22px 6px;
```

Comment:
One of the reasons that we use `@xstyled` is because we don’t need to add this callback so many times. It simplifies the code and forces us to always get values from the theme itself.

```suggestion
  background-color: material; // this will get material from theme.colors
  color: materialText; // this will get materialText from theme.colors
  padding: 6 22 6; // this will get the 6th and 22th from theme.spacing
```